lib/players/human.rb

```diff
@@ -0,0 +1,7 @@
+class Player::Human < Player
+  def move(board=nil)
+    puts "Hello, Player #{token}, what is your move(1-9)?"
+    move = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```